vendor/assets/stylesheets/dvl/core/forms.scss

```diff
@@ -327,6 +327,25 @@ input[type="radio"] {
   }
 }
 
+.input_group_date,
+.input_group_time {
+  float: left;
```

Comment:
I don't think this is correct. We should only float them to the left in the context of a `datetime_input_wrapper`.